debugtools/DDR_VM/src/com/ibm/j9ddr/CompatibilityConstants29.dat

```diff
@@ -92,3 +92,5 @@ MM_ForwardedHeader.OMR_FORWARDED_TAG_MASK = 0
 ShcdatatypesConstants.TYPE_PREREQ_CACHE = -1
 
 ShCFlags.J9SHR_DATA_TYPE_STARTUP_HINTS = -1
+
+J9SFStackFrame.J9SF_CompactVectorState = 0
```

Comment:
Please move this up before `MM_ForwardedHeader` so the list remains sorted by class.